config.toml

```diff
@@ -45,10 +45,10 @@ bootnodes = [] # example : spacemesh://j7qWfWaJRVp25ZsnCu9rJ4PmhigZBtesB4YmQHqqP
 
 # API Config
 [api]
-grpc-server = true
+grpc = "node" # Only run the Node service by default
 json-server = true
-grpc-port = 9091
-json-port = 9090
+grpc-port = 9092
```

Comment:
why change the port?

Reply:
We don't need to change the port, but the API services are totally different, so it's sort of a way of requiring any clients (tests, etc.) that were talking to the old API service to explicitly choose to talk to the new one, does that make sense? Let me know what you think.